intro-to-debugging/intro-to-debugging.md

```diff
@@ -77,14 +77,6 @@ When we come across errors, we should take notes of three things:
 2. What is the name of the error
 3. What is the line number of the error
 
-<!-- ![Screenshot of two windows: One is a .rb file with a bug, and one is Terminal running the file, and showing a NameError](images/intro_debugging-stack_trace.png) -->
-
-From this example, our answers are:
```

Comment:
A student pointed out that this sentence is here but there is no example and there are no answers because they were commented out. Deleting the sentence and commented out portion.